geocity/apps/submissions/models.py

```diff
@@ -1727,6 +1728,12 @@ class Meta:
 
 class SubmissionAmendField(models.Model):
     name = models.CharField(_("nom"), max_length=255)
+    api_name = models.CharField(
+        _("Nom dans l'API"),
+        max_length=255,
+        blank=True,
+        help_text=_("Se génère automatiquement lorsque celui-ci est vide."),
+    )
```

Comment:
Should be unique to avoid within a form in order to avoid duplicate key in json ?